test/Microsoft.Health.SqlServer.Web/Properties/launchSettings.json

```diff
@@ -3,11 +3,7 @@
     "Microsoft.Health.SqlServer.Web": {
       "commandName": "Project",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "development",
```

Comment:
@rbans96 , I removed these environment variables to disable the `SchemaInitializer` from running in the E2E test server. Previously, the schema initializer did not run because we did not register the `InitializationModule`, so `IStartable`s were not started. Now with change, `IHostedService`s *will* be started. When running in Github actions, we do not have a database to connect to at the given connection string so it fails.

Reply:
Thanks for the information.